src/index.js

```diff
@@ -206,68 +278,7 @@ module.exports = (
         }
       ]
     },
-    plugins: [
-      {
-        apply(compiler) {
```

Comment:
this was just moved above so we can dynamically add the `LicenseWebpackPlugin` to this array based on the results of the `license` flag